rdf/ebuccdm.owl

```diff
@@ -22,8 +22,8 @@
                                                                 "Jean-Pierre Evain, EBU"@ch ,
                                                                 "Jürgen Grupp, SWR"@de ,
                                                                 "Kim Viljanen, YLE"@fi ,
-                                                                "Masoud Aminipour, not affiliated" ,
-                                                                "Tormod Værvågen, NRK"@nb ;
```

Comment:
Why do you propose to remove my name, @alexandre-rouxel

Reply:
Of course, I didn't want to remove it. Apologies,  false manipulation, I did not remove any name. 
Thanks for detecting it.

Reply:
No - problem, the real problem I was looking at is there is some conflict somewhere in the file, so merging the branch can not be done automatically. 
Don't worry, I will have a look.